resources/locales/fr/translation.json

```diff
@@ -38,5 +34,27 @@
   "oben links": "en haut à gauche",
   "oben rechts": "en haut à droite",
   "unten links": "en bas à gauche",
-  "unten rechts": "en bas à droite"
+  "unten rechts": "en bas à droite",
+  "Punkt": "Point",
+  "Punktumrahmung": "Contour du point",
+  "Punktumrahmung (prominent)": "Contour du point (important)",
+  "Land": "Pays",
+  "Hauptstadt": "Capitale",
+  "Stadt": "Ville",
+  "Gewässer": "Eau",
+  "Gebiet": "Région",
+  "Ort (ungenau)": "Lieu (incertain)",
+  "Ereignis": "Événement",
+  "Epizentrum": "Épicentre",
+  "Explosion": "Explosion",
+  "Feuer": "Feu",
+  "Flagge": "Drapeau",
+  "Flughafen": "Avion",
```

Comment:
see comment in english translation

Reply:
Thanks for the notice. I have added the translations in https://github.com/nzzdev/Q-locator-map/pull/59/commits/af9ec2d570884a818ad26c29d969a7ef4a04ac16